mirrord/cli/src/execution.rs

```diff
@@ -504,8 +505,8 @@ impl MirrordExecution {
             env_vars.extend(envs_from_file);
         }
 
-        if let Some(overrides) = &config.feature.env.r#override {
-            env_vars.extend(overrides.iter().map(|(k, v)| (k.clone(), v.clone())));
+        if let Some(mapping) = config.feature.env.mapping.clone() {
+            env_vars = EnvVarsRemapper::new(mapping, env_vars).remapped();
```

Comment:
Why remove overrides? :eyes:

Reply:
Argh, think I wrote over it when copy pasting the `if let`. Nice catch.